apis/v1alpha1/greptimedbcluster_types.go

```diff
@@ -18,11 +18,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// ComponentSpec is the common specification for all components(frontend/meta/datanode).
+// ComponentSpec is the common specification for all components(`frontend`/`meta`/`datanode`/`flownode`).
 type ComponentSpec struct {
 	// The number of replicas of the components.
-	// +optional
-	// +kubebuilder:validation:Minimum=0
+	// +required
+	// +kubebuilder:validation:Minimum=1
```

Comment:
I think the replicas minimum is 0, and sometimes it is necessary to set the replicas count to 0 to disconnect the frontend network connection.